about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Holi 2023</title>
+  <link rel="stylesheet" href="./css/about.css">
+  <link rel="stylesheet" href="./css/index.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+</head>
+<body>
+  <header>
+    <div class="topbar">
+      <ul>
+        <li><a href="#"><i class="bi bi-facebook"></i></a></li>
+        <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <nav>
+      <div class="nav-bar">
+        <a href="index.html"><img src="./images/Logo.png" alt="holi-logo" class="logo"></a>
+        <button class="hambug-menu">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+        <ul class="mobile-menu">
+          <li><a class="menuItem" href="index.html">Home</a></li>
+          <li><a class="menuItem" href="about.html">About</a></li>
+          <li><a class="menuItem" href="#">Programs</a></li>
+          <li><a class="menuItem" href="#">Gallery</a></li>
+          <li><a class="menuItem" href="#">Come Celebrate</a></li>
+        </ul>
+
+        <ul class="nav-links">
+          <li class="links"><a class="alinks" href="about.html">About</a></li>
+          <li class="links"><a class="alinks" href="#">Programs</a></li>
+          <li class="links"><a class="alinks" href="#">Gallery</a></li>
+          <li class="links"><a class="alinks borderLink" href="#">Come Celebrate</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  
+  <section class="aboutHeadContent">
+    <h1 class="aboutHeaderTitle">"Have a Happy Holi!"</h1>
+    <h2 class="aboutSubHeader">HOLI - "Festival of Colors" or the "Festival of Love"</h2>
+    <p class="aboutHeaderParagraph">Holi has religious and mythological significance, marking the victory of good over evil and the arrival of spring. Holi is now celebrated globally by people of Indian origin and has become a symbol of India's cultural richness and diversity.</p>
```

Comment:
- [ ] Dear @Kajal19-del, I notice you would like to stick to your current background color from your additional information for the PR, But Please understand we are trying to parse a design and as a front-end developer it's a must-have skill to be able to parse a design into its exact copy. Please follow the background as specified in the design guideline for both mobile and desktop views.
- [ ] Please, Please follow the background for the mobile display too.
### Figma design 
![217946836-201d5bdb-1186-462e-9019-72e7dcfc6e12](https://user-images.githubusercontent.com/69077061/218030315-55339d45-f606-4c3c-ba50-d1ac7df80987.png)

### Your design 

![217946694-9389155d-74ca-4412-ae56-cde68990101e](https://user-images.githubusercontent.com/69077061/218030355-003eb084-ab7f-4fb1-98bb-4d404c475568.png)